atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -243,20 +244,30 @@ def set_group_manifest_info(self, extra, worker_metadatas):
             repo = ImageName.parse(repositories[0]).to_str(registry=False, tag=False)
             # group_manifests added the registry, so this should be valid
             registries = self.workflow.push_conf.all_registries
+            digest_version = None
+            digest = None
+            # try to find the actual str digest value in the manifest digest
+            try:
+                for digest_version, digest in manifest_digest.default.items():
+                    break
+            # the manifest digest isn't a manifest digest, but a simple dict
+            except AttributeError:
+                for digest_version, digest in manifest_digest.items():
+                    if digest:
+                        break
```

Comment:
This doesn't look right. `manifest_digest.default` is a method.

It's a shame ManifestDigest.default() only returns the digest, where we also want the type. Ideally there would be a .default_type_and_digest() which would tell us both.

@MartinBasti @rcerven @athos-ribeiro thoughts?

Reply:
@twaugh  group_manifest plugin should always return that digest as ManifestDigest even for non-v2list
then we don't need this logic at all

Reply:
Yes, it should always be a ManifestDigest instance. But what's a nice way to get the digest and its content type from one of those? This 'for digest_version, digest in manifest_digest.items()' ok? It makes the assumption there is a single digest -- are we happy with that?

Alternative is to use manifest_digest.default() for the digest, then match that with a type.

Reply:
I don't understand issue here.  `default` returns default schema, so I can get digest from that:
```python
digest = manifest_digest[manifest_digest.default]
```

Reply:
Oh, `.default` is a *property*... sorry, I'd misread it as a method.

But it returns the digest, not the content type.

Reply:
Ah nevermind, I read that code wrongly

Reply:
Maybe we can add property `default_media_type`

Reply:
the __getattr__ logic in ManifestDigest frustrated my attempts to make this less ugly.

Reply:
@mlangsdorf so could we add a `default_media_type` property to ManifestDigest, which returns the abbreviated type name (e.g. 'v2_list') of the first digest value in the preference order list?

Both `default` and `default_media_type` should share the same preference order list, so it might make sense to have it defined in a class attribute.

Then this code can simply be
```
digest_version = manifest_digest.default_media_type
digest = manifest_digest[digest_version]
```

Reply:
Fixed, and the errant test case changed.